tests/GihtubAPI.feature

```diff
@@ -0,0 +1,63 @@
+# Created by dvlad at 12/19/2023
+Feature: GitHub API
+  # Enter feature description here
+
+  Scenario: Get all repos
+    When Send GET request to https://api.github.com/search/repositories
+      | q | webdrivercamp-learning-python |
+    And Verify response value
+      | status code | 200       |
+      | No of Repos | 23        |
+      | name        | dvlad7909 |
+
+  Scenario: Get repo with authentication
+    When Send GET request with authentication to https://api.github.com/user/repos
+    Then Verify response value
+      | status code | 200 |
+      | No of Repos | 11  |
+
+  Scenario: Create Gist
+    Given Load payload from gist_payload.json file with updated values
+      | $.description | Gist created with api 2 |
+      | $.public      | true                    |
+    When Send POST request with authentication to https://api.github.com/gists
+      | scope | gist |
+    And Save gist id
+    Then Verify response value
+      | status code | 201 |
+
+  Scenario: Delete Gist
+    When Send DELETE Gist request with authentication to https://api.github.com/gists/
+    Then Verify response value
+      | status code | 204 |
+
+  Scenario: Create Repo
+    Given Load payload from repo_payload.json file with updated values
+      | $.name     | repo-created-with-api |
```

Comment:
Yes you are right. It will be better to add that '$.' in a code. 
I dodn't know why you can't see my /base repo it's public, but I added you as collaborator to /base repo, just in case.

Reply:
yeah, I see it now. It's just not really convenient to check it because it is not a part of this pull request. I will check later..

Reply:
I have same issue in your project. Maybe there are some settings for submodules...

Reply:
Done